.github/workflows/test_win2019.yml

```diff
@@ -0,0 +1,59 @@
+name: Win2019 tests
+
+on: [push]
+
+jobs:
+  test:
+
+    runs-on: windows-2019
+
+    strategy:
+      fail-fast: false
+      matrix:
+        # group modules to get a job runtime of ~15min
+        module: [
+            datalad.core,
+            datalad.local datalad.distributed datalad.support,
+            datalad.customremotes datalad.downloaders datalad.plugin,
+            datalad.distribution,
+            datalad.interface,
+            datalad.metadata datalad.tests datalad.ui datalad.cmdline
+        ]
+    steps:
+    - name: Set up environment
+      run: |
+        git config --global user.email "test@github.land"
+        git config --global user.name "GitHub Almighty"
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Set up git-annex
+      run: |
+        powershell.exe Import-Module BitsTransfer; Start-BitsTransfer -Source https://downloads.kitenet.net/git-annex/windows/current/git-annex-installer.exe -Destination C:\\git-annex-installer.exe
+        7z x -o"C:\\Program Files\Git" C:\\git-annex-installer.exe
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install colorama
+        pip install ".[tests]"
+        pip install ".[devel-utils]"
+    - name: WTF!?
+      run: |
+        datalad wtf
+        dir
+    - name: ${{ matrix.module }} tests
+      run: |
+        mkdir -p __testhome__
+        cd __testhome__
+        python -m nose -s -v --with-cov --cover-package datalad ${{ matrix.module }}
+    # coverage report is not functional because codecov refuses to accept the
+    # report
```

Comment:
eh heh -- that is sad.
From all the annotation of tests I was not sure if any of those was passing on appveyor but fails on github ci?

Reply:
I would be surprised if that would be the case, but I cannot claim that with much confidence. In any case, this is not meant to replace appveyor in the short term. My experience is that windows systems are highly flexible.Many things that work on appveyor fail on my win10 test box. This just gives us a better grasp on real-world variability.